app/_kong_plugins/datadog/index.md

```diff
@@ -29,4 +29,145 @@ categories:
+`latency`                  | Tracks the interval between the time the request started and the time the response was received from the upstream server | `kong.latency`
+`upstream_latency`         | Tracks the time it took for the final service to process the request | `kong.upstream_latency`
+`kong_latency`             | Tracks the internal Kong latency that it took to run all the plugins | `kong.kong_latency`
+
+The metrics will be sent with the tags `name` and `status` carrying the API name and HTTP status code respectively. If you specify `consumer_identifier` with the metric, a `consumer` tag will be added.
+
+All metrics get logged by default. You can customize the metrics logged with the [`config.metrics`](./reference/#schema--config-metrics) parameter. Note that metrics with `stat_type` set to `counter` or `gauge` must have `sample_rate` defined as well.
+
+## Migrating Datadog queries
+The plugin updates replace the API, status, and consumer-specific metrics with a generic metric name.
+You must change your Datadog queries in dashboards and alerts to reflect the metrics updates.
+
+For example, the following query:
+```
+avg:kong.sample_service.latency.avg{*}
+```
+would need to change to:
+
+```
+avg:kong.latency.avg{name:sample-service}
+```
+
+## Setting host and port per Kong node
+
+When installing a multi-data center setup, you might want to set Datadog's agent host and port for each Kong node. This configuration is possible by setting the host and port properties with environment variables.
+
+Field           | Description                                           | Data types
+---             | ---                                                   | ---
+`KONG_DATADOG_AGENT_HOST` | The IP address or hostname to send data to | string
+`KONG_DATADOG_AGENT_PORT` | The port to send data to on the upstream server | integer
+
+{:.info}
+> **Note:** The `host` and `port` fields in the plugin configuration take precedence over environment variables.
+> For Kubernetes, there is a known limitation that you can't set `host` to null to use the environment variable. 
+> You can work around this by using a vault reference, for example: `{vault://env/kong-datadog-agent-host}`. 
+> Refer to [Configure with Kubernetes](#configure-with-kubernetes).
+
+## Kong process errors
+
+{% include /plugins/logging/kong-process-errors.md %}
+
+
+## Configure with Kubernetes
+
+In most Kubernetes setups, `datadog-agent` runs as a daemon set. 
+This means that a `datadog-agent` runs on each node in the Kubernetes cluster, and {{site.base_gateway}} must forward metrics to the `datadog-agent` running on the same node as {{site.base_gateway}}. 
+
+This can be accomplished by providing the IP address of the Kubernetes worker node to {{site.base_gateway}}, then configuring the plugin to use that IP address using environment variables.
+
+{% navtabs "Kubernetes" %}
+{% navtab "Helm" %}
+
+1. Modify the `env` section in `values.yaml`:
```

Comment:
```suggestion
Modify the `env` section in `values.yaml`:
```
I think the numbered steps here felt a bit off since the instructions aren't end to end, so I think removing them could help this feel more like a reference.